docs/data/charts/sparkline/CustomDomainYAxis.js

```diff
@@ -11,21 +11,51 @@ const settings = {
   showHighlight: true,
 };
 
-const values = [60, -25, 66, 68, 87, 82, 83, 100, 92, 75, 76, 50, 91];
+// Extend a value to match a multiple of the step.
+function extend(value, step) {
+  if (value > 0) {
+    // If >0 go to the next step
+    return step * Math.ceil(value / step);
+  }
+  // If <0 go to the previous step
+  return step * Math.floor(value / step);
+}
+
+const values = [60, -15, 66, 68, 87, 82, 83, 85, 92, 75, 76, 50, 91];
 
 export default function CustomDomainYAxis() {
   return (
-    <Stack sx={{ width: '100%' }}>
-      <Typography>Without strict domain limit</Typography>
-      <Stack sx={{ width: '100%', mb: 2 }} direction="row" spacing={2}>
+    <Stack
+      sx={(theme) => ({
+        width: '100%',
+        '& p': { mb: 1, mt: 2 },
+        '& svg': {
+          borderWidth: 1,
+          borderStyle: 'solid',
+          borderColor: theme.palette.divider,
```

Comment:
I added a border to the SVGs such that we can more easily the charts from the layout spacings